silkworm/rpc/http/connection.cpp

```diff
@@ -45,13 +46,8 @@ Connection::Connection(boost::asio::io_context& io_context,
                        std::optional<std::string> jwt_secret)
     : socket_{io_context},
       request_handler_{this, api, handler_table},
-      buffer_{},
       allowed_origins_{allowed_origins},
       jwt_secret_{std ::move(jwt_secret)} {
-    request_.content.reserve(kRequestContentInitialCapacity);
-    request_.headers.reserve(kRequestHeadersInitialCapacity);
-    request_.method.reserve(kRequestMethodInitialCapacity);
-    request_.uri.reserve(kRequestUriInitialCapacity);
```

Comment:
The definitions of all these constants (here and in `constants.hpp`) should be removed as they are not used anymore.

Reply:
ok